GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -31,7 +31,7 @@
 #include "../gl/gl_program.h"
 
 namespace gvr {
-Mesh* Mesh::getBoundingBox() {
+Mesh* Mesh::createBoundingBox() {
 
```

Comment:
You can give it a try. You need to check for code that assumes it is getting a new instance and manages this instance's lifecycle.